pkg/controller/manager/sync_action.go

```diff
@@ -201,6 +204,19 @@ func syncRepairTasks(clusterID string, cluster *scyllav1.ScyllaCluster, syncer s
 	return actions, nil
 }
 
+type startDateGetSetter interface {
```

Comment:
nit: `startDateGetterSetter` or `startDateInterface`
on the other hand would it work with just inheritance?

Reply:
renamed, would you mind showing a pseudocode for what you have in mind regarding last part?